librz/arch/swift_rtti.c

```diff
@@ -0,0 +1,83 @@
+// SPDX-FileCopyrightText: 2025 tushar3q34 <tushar3q34@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include "rz_analysis.h"
+#include "rz_core.h"
+#include "rz_io.h"
+
+static void rz_add_metaclass_info(HtUP *swift_metaclass_info, RzList *fields, char *class_name) {
```

Comment:
if it is a `static` function, maybe avoid having the `rz_` prefix on it. 
The idea is that RZ_API & RZ_IPI are the "exposed" ones thus the functions will always start with `rz_` prefix